### PR TITLE
Excluded indexes

### PR DESCRIPTION
**Pull request by iamalisalehi**
New PR to implement changes requested by @J3ldo and my suggestions discussed in PR #105 .
This PR also resolves Issue #29 .

### CODE REVIEW

**Comment by iamalisalehi** [comment]
Hi @aisk , a couple of questions:
Should I write a new test for excluded indexes? What about a new example?

File: src/pick/__init__.py [hunk 1]
```diff
@@ -12,6 +12,7 @@ class Option:
     label: str
     value: Any = None
     description: Optional[str] = None
+    enable: bool = True
```
Comment (aisk) [inline src/pick/__init__.py:15]: I prefer `enabled`, just like https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/disabled

**Comment by aisk** [comment]
> Hi @aisk , a couple of questions: Should I write a new test for excluded indexes? What about a new example?

Thanks for the PR! Test and example will be appreciated.

**Comment by iamalisalehi** [comment]
Thanks for the comment @aisk , I think the PR is better now, but I'm not sure whether I raised the right error on [line 59](https://github.com/aisk/pick/blob/c422252f7afad3aea726563fea17bf20e118e742/src/pick/__init__.py#L59) or not.

**Comment by aisk** [comment]
> Thanks for the comment @aisk , I think the PR is better now, but I'm not sure whether I raised the right error on [line 59](https://github.com/aisk/pick/blob/c422252f7afad3aea726563fea17bf20e118e742/src/pick/__init__.py#L59) or not.

As Python's document said https://docs.python.org/3/library/exceptions.html#ValueError:

> Raised when an operation or function receives an argument that has the right type but an inappropriate value, and the situation is not described by a more precise exception such as [IndexError](https://docs.python.org/3/library/exceptions.html#IndexError).

`ValueError` is the best choice in this case.

File: example/disabled.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
+from pick import pick, Option
+
```
Comment (aisk) [inline example/disabled.py:2]:
```suggestion


```

File: src/pick/__init__.py [hunk 3]
```diff
@@ -54,17 +55,33 @@ def __post_init__(self) -> None:
                 "min_selection_count is bigger than the available options, you will not be able to make any selection"
             )
 
+        if all(isinstance(option, Option) and option.enabled == False for option in self.options):
```
Comment (aisk) [inline src/pick/__init__.py:58]:
```suggestion
        if all(isinstance(option, Option) and not option.enabled for option in self.options):
```

**Review by aisk — COMMENTED** [review]
LGTM, I only have some nitpicks:

**Comment by aisk** [comment]
Thanks for contributing!

**Pull request merged**